examples/hosts/app-integration/external-controller/README.md

```diff
@@ -42,3 +42,26 @@ For in browser testing update `./jest-puppeteer.config.js` to:
 Dice Roller uses the following distributed data structures:
 
 - SharedDirectory - root
+
+## Backed Locally and running with live FRS instance
+
+We can connect to both live FRS instance by passing in the tenanID, orderer and storage as well as using the tenantID as "local" for running against Tinylicious for development purpose.
```

Comment:
Let's have a uniform way of talking about "tenant ID". Whenever we refer to as the value itself, we should call it with the space, and if we are specifying it as the parameter in code, lets have it as `tenantId` in a code block